nix_update/__init__.py

```diff
@@ -178,8 +178,7 @@ def main() -> None:
     if not os.path.exists(options.import_path):
         die(f"path {options.import_path} does not exists")
 
-    if options.commit:
-        git_dir = validate_git_dir(options.import_path)
+    git_dir = validate_git_dir(options.import_path)
```

Comment:
This basically assumes that there everyone uses git, which might be not the case.